.gitignore

```diff
@@ -9,3 +9,6 @@ target/
 /scripts/closePR
 /scripts/publishAndTag
 /scripts/readVersion
+
+*.swo
+*.swp
```

Comment:
No harm adding these but FYI you can use `~/.gitignore_global` for editor specific ignores.